application/src/uuidmanager.cpp

```diff
@@ -3,34 +3,51 @@
 #include <QJsonDocument>
 #include <QJsonObject>
 #include <QJsonParseError>
+#include <QStandardPaths>
 #include <QFile>
 #include <QDir>
 
-static const QString FILE_PATH = QDir::homePath() + "/UUID.txt";
 
-UUIDManager::UUIDManager(QObject *parent)
+static const QString FILE_PATH = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + "/rheda-messenger";
```

Comment:
```suggestion
static const QString CONFIG_PATH = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + "/rheda-messenger/";
```
static const QString FILE_PATH = CONFIG_PATH + "user.conf"

Reply:
я думаю тут нужна еще одна константа для самого файла